### PR TITLE
[ADD] mail-addons 9.0 mail_reply

### PR DESCRIPTION
**Pull request by x620**

### CODE REVIEW

File: mail_reply/README.rst [hunk 1]
```diff
@@ -0,0 +1,16 @@
+===================
+ Show reply button
+===================
+
+The module adds the reply button on the message is not attached to the recording.
```
Comment (yelizariev) [inline mail_reply/README.rst]: The module allows to reply to any message. Out-of-box odoo shows reply button for messages attached to some record.

File: mail_reply/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+{
+    "name": """Show reply button""",
+    "summary": """The module adds the reply button on the message is not attached to the recording""",
```
Comment (yelizariev) [inline mail_reply/__openerp__.py]: put a slogan here

Comment (yelizariev) [inline mail_reply/__openerp__.py]: e.g.

    Got a message out of Form? Now you can reply to it too!

File: mail_reply/doc/index.rst [hunk 3]
```diff
@@ -0,0 +1,9 @@
+===================
+ Show reply button
+===================
+
+Usage
+=====
+
+* Open Discuss menu
+* All messages is not attached to the recording have reply button
```
Comment (yelizariev) [inline mail_reply/doc/index.rst]: could be shorten

    * All messages have reply button

File: mail_reply/__openerp__.py (hunk 2 above)
Comment (yelizariev) [inline mail_reply/__openerp__.py]: Got a message out of a  Form? Now you can reply to it too!

**Comment by x620** [comment]
Дальнейшая разработка велась здесь: https://github.com/yelizariev/mail-addons/pull/33

**Pull request closed without merging**